cartographer_grpc/framework/server_test.cc

```diff
@@ -28,9 +30,11 @@ namespace cartographer_grpc {
 namespace framework {
 namespace {
 
+using EchoCallback = std::function<bool()>;
```

Comment:
naming: This is a writer action with a captured response, name it   WriteResponseFunction, WriteResponseCallback?

Reply:
Renamed to `EchoResponder`.